### ISSUE TITLE
[Request] farm input improvements

### ISSUE THREAD

**Issue by Terpo** [ForestryMC/ForestryMC#1518]
I really like the automated farms of forestry, but there a few things that are a few things that can be annoying. 

The intention to place items in the output slot is fully ok, but the input has some problems. 

For example you want to create fancy managed vegetable farm.

![farm](https://cloud.githubusercontent.com/assets/4256058/20685620/62cc0582-b5b5-11e6-9d49-c6257296415e.PNG)

Your intention is: One side for wheat, one for carrots, one for potatoes and one for beetroot. Problem here is, that you have no control where the crops are placed. Only wheat seeds will find their way always in the right spot. Carrots, potatoes and beetroot will be planted more or less random. Helpful would be some kind of lock switch for the input. In normal mode it will work as it do at the moment. But in locked mode the 12 input slots will get seperated and "maybe" do not use the last item in the slot. (or put in a ghost item to remember it)
So, there are 12 slots.... 4 for the soil, 2 for each orientation.

![farm2](https://cloud.githubusercontent.com/assets/4256058/20686096/877667d6-b5b7-11e6-9228-b50f8b323402.PNG)

Another thing is #1494 buts thats another topic :)

**Comment by Nedelosk** [comment]
The crop logic now automatically replants the crop after it was harvested. So this is no longer needed.

**Comment by tornewuff** [comment]
Hm. This change broke my farms and I only just noticed when playing on a modpack that has a newer version of Forestry. I have a bunch of automation I usually build that feeds different crop seeds into the multifarm depending on the stock levels of the produce they grow into, such that it would grow the things I was short of, but now it just keeps growing whatever it happens to plant first forever, and doesn't "switch over" when I've got plenty.

This seems like it makes automatic farms less useful.. if you want to grow multiple crops you might as well plant them all by hand in whatever ratio you want and then just use a manual farm now (unless I'm misunderstanding the effect of the change here; I haven't tried that)

Having it automatically replant definitely makes simple use cases easier, though, so maybe this is a good tradeoff; just wanted to bring an alternative perspective here :)
